language-support/codegen-common/src/main/scala/com/digitalasset/daml/lf/codegen/conf/CodegenConfigReader.scala

```diff
@@ -140,7 +139,6 @@ object CodegenConfigReader {
 
   private def dest(a: CodegenDest): String = a match {
     case Java => "java"
-    case Scala => "scala"
```

Comment:
this isn’t going to fly. Canton currently consumes this JAR to run the Scala codegen. We either need to release a standalone JAR using just the Scala codegen or leave this for now.

Reply:
Thanks for making me aware of this. I'll add the standalone JAR as part of this PR. Would you mind blocking it to avoid accidental merges? I already have a couple of approvals and I can just dismiss your block once I did the work to have the standalone JAR. 🙇🏻

Reply:
Addressed by https://github.com/digital-asset/daml/pull/12167/commits/8a03f8faa108fb416176d941aee0d1b41b28f8d8